README.md

```diff
@@ -1,16 +1,16 @@
-# u8timeseries: Easy manipulation and forecasting of time series
+# darts: Easy manipulation and forecasting of time series
 
-![u8timeseries](https://github.com/unit8co/u8timeseries/raw/develop/static/images/logo-72dpi.png "u8timeseries") 
+![darts](https://github.com/unit8co/darts/raw/develop/static/images/logo-72dpi.png "darts") 
 
 ---
-[![PyPI version](https://badge.fury.io/py/u8timeseries.svg)](https://badge.fury.io/py/u8timeseries)
-![GitHub Workflow Status](https://img.shields.io/github/workflow/status/unit8co/u8timeseries/develop)
+[![PyPI version](https://badge.fury.io/py/darts.svg)](https://badge.fury.io/py/darts)
+![GitHub Workflow Status](https://img.shields.io/github/workflow/status/unit8co/darts/develop)
 ![Supported versions](https://img.shields.io/badge/python-3.6+-blue.svg)
-![Docker Image Version (latest by date)](https://img.shields.io/docker/v/unit8/u8timeseries?label=docker&sort=date)
-![PyPI - Downloads](https://img.shields.io/pypi/dm/u8timeseries)
-![GitHub Release Date](https://img.shields.io/github/release-date/unit8co/u8timeseries)
+![Docker Image Version (latest by date)](https://img.shields.io/docker/v/unit8/darts?label=docker&sort=date)
+![PyPI - Downloads](https://img.shields.io/pypi/dm/darts)
+![GitHub Release Date](https://img.shields.io/github/release-date/unit8co/darts)
```

Comment:
We need to change repo name as well

Reply:
yep, the question is before or after merging - in both cases I think something will break ;]

Reply:
let's try to change the repo name after merging